### PR TITLE
Fix test coverage report

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Fix test coverage report.

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/21437

### CODE REVIEW

File: packages/dashboard-frontend/src/store/Plugins/devWorkspacePlugins/__tests__/index.spec.ts [hunk 1]
```diff
@@ -201,11 +201,12 @@ describe('dwPlugins store', () => {
...
-        await store.dispatch(dwPluginsStore.actionCreators.requestDwEditor(settings, editorLink));
-      } catch (e) {
-        // noop
-      }
+      await store
+        .dispatch(dwPluginsStore.actionCreators.requestDwEditor(settings, editorLink))
+        .catch(() => {
+          // noop
+        });
+
```
Comment (akurinnoy) [inline packages/dashboard-frontend/src/store/Plugins/devWorkspacePlugins/__tests__/index.spec.ts:209]: Please revert these lines, changes are not related to the issue.

**Review by akurinnoy — APPROVED** [review]
Good work!

**Review by ibuziuk — APPROVED** [review]
